File: in_go/src/pattern_11.go
package src

import "fmt"

func Pattern_11() {
	row := 9
	column := 5
	symbol := "*"
	halfRow := row / 2

	for i := 0; i < row; i++ {
		for j := 0; j < column; j++ {
			if i <= halfRow {
				if j <= i {
					fmt.Print(symbol)
				} else {
					fmt.Print(" ")
				}
			} else {
				tempI := row - 1 - i
				if j <= tempI {
					fmt.Print(symbol)
				} else {
					fmt.Print(" ")
				}
			}
		}
		fmt.Println()
	}
}

File: in_rust/src/pattern_11.rs
pub fn pattern_11() {
    let row = 9;
    let column = 5;
    let symbol = "*";
    let half_row = row / 2;

    for i in 0..row {
        for j in 0..column {
            if i <= half_row {
                if j <= i {
                    print!("{}", symbol);
                } else {
                    print!(" ");
                }
            } else {
                let temp_i = row - 1 - i;
                if j <= temp_i {
                    print!("{}", symbol);
                } else {
                    print!(" ");
                }
            }
        }
        println!();
    }
}
